spec/factories/tasks.rb

```diff
@@ -1,9 +1,14 @@
 FactoryBot.define do
   factory :task do
-    title { 'Task' }
+    title  { 'Task' }
     status { rand(2) }
     from = Date.parse("2019/08/01")
     to   = Date.parse("2019/12/31")
     deadline { Random.rand(from..to) }
+    
+    trait :done do
+      status { "done" }
```

Comment:
シンボル形式で定義してください。String型でないことが明示できます。
```suggestion
      status { :done }
```

Reply:
ea222d0　シンボルにて定義しました。
以前も指摘してもらった事なので意識します。　